tasks/aiida-prepare.yml

```diff
@@ -37,11 +66,24 @@
     line: "export VIRTUALENVWRAPPER_PYTHON={{ ansible_python_interpreter }}"
   when: ansible_python_interpreter is defined
 
-- name: Install pip packages
+- name: Install pip packages globally
+  become: true
+  become_user: "{{ root_user }}"
+  pip:
+    name:
+      - virtualenvwrapper
+
+- name: Install pip packages in the AiiDA virtual environment
   pip:
     name:
       - pip~=19.3
       - numpy==1.16.4
+      # Flask is required for the AiiDA REST API
```

Comment:
That should be taken care of by installing the `rest` extra of the AiIDA package

Reply:
Somehow it did not. Removing now and then we will see how it goes.

Reply:
looking at where the git repo is installed, it seems to me we're simply not installing the `rest` extra

Reply:
Aha, tests are running fine now so that was probably added when one of the plugins was enabled. Maybe. By the way do we want to have the plugin installation enabled by default?

Reply:
Ok, I see... I'm fine with disabling it by default. 

This aiida role really started as a role to install the aiida environment for the Quantum Mobile. Eventually I might move out the plugins/codes/examples to a separate role.

Reply:
As long as it is in this role, however, we should keep testing it (i.e. please keep it in the molecule playbook)